src/connectors/range/connectRange.js

```diff
@@ -305,43 +300,39 @@ export default function connectRange(renderFn, unmountFn = noop) {
       },
 
       getWidgetSearchParameters(searchParameters, { uiState }) {
-        const value = uiState && uiState.range && uiState.range[attribute];
+        let withFacetConfiguration = searchParameters.setQueryParameters({
+          disjunctiveFacets: [attribute],
+          numericRefinements: {
+            ...searchParameters.numericRefinements,
+            [attribute]: {},
+          },
+        });
+
+        const value = uiState.range && uiState.range[attribute];
 
         if (!value || value.indexOf(':') === -1) {
-          return searchParameters;
+          return withFacetConfiguration;
         }
 
-        const {
-          '>=': previousMin = [NaN],
-          '<=': previousMax = [NaN],
-        } = searchParameters.getNumericRefinements(attribute);
-        let clearedParams = searchParameters.clearRefinements(attribute);
         const [lowerBound, upperBound] = value.split(':').map(parseFloat);
 
-        if (
-          previousMin.includes(lowerBound) &&
```

Comment:
makes sense to remove this, and rely on the helper to avoid duplicates. does this have a test?